Index.html

```diff
@@ -0,0 +1,282 @@
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
+            <p class="text-content-2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="btn-list-1">
+              <li>
+                <button class="btn-html-1"><a href="#">HTML</a></button>
+              </li>
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
+            <p class="text-content-2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="btn-list-1">
+              <li>
+                <button class="btn-html-1"><a href="#">HTML</a></button>
+              </li>
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+          <div class="pro-art">
+            <h1 class="headline-4">Profesional Art Printing Data</h1>
+            <p class="text-content-2">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="btn-list-1">
+              <li>
+                <button class="btn-html-1"><a href="#">HTML</a></button>
+              </li>
+              <li>
+                <button class="btn-bootstrap-1">
+                  <a href="#">BOOTSTRAP</a>
+                </button>
+              </li>
+              <li>
+                <button class="btn-ruby-1"><a href="#">RUBY</a></button>
+              </li>
+            </ul>
+            <button class="btn-sub-2"><a href="#">See project</a></button>
+          </div>
+        </main>
+      </section>
```

Comment:
- [ ] As per the given instruction you need to use `display: grid` to layout the cards. Kindly fix this issue in CSS file.